lib/remote/jsonrpcconnection.cpp

```diff
@@ -84,7 +84,7 @@ void JsonRpcConnection::HandleIncomingMessages(boost::asio::yield_context yc)
 
 			l_TaskStats.InsertValue(Utility::GetTime(), 1);
 		} catch (const std::exception& ex) {
-			Log(m_ShuttingDown ? LogDebug : LogWarning, "JsonRpcConnection")
+			Log(LogWarning, "JsonRpcConnection")
```

Comment:
The previous condition was `m_ShuttingDown ? Debug : Warning`. Since this code part should no longer be executed while `m_ShuttingDown` is set, it must be a warning as it was originally.

Reply:
It _could_ be executed while m_ShuttingDown is set. E.g m_ShuttingDown gets set just after successfully reading a message.

Reply:
Although `m_ShuttingDown` might be set meanwhile being here, the coroutine from `Disconnect()` should not run in parallel with this one, meaning the socket should still be functional. But I'll leave it as it was before!